lib/chef/provider/mount/mount.rb

```diff
@@ -146,6 +146,15 @@ def remount_fs
           end
         end
 
+        # Return appropriate default mount options according to the input os.
+        def default_mount_options(os)
+          if os == "freebsd"
+            "rw"
+          else
+            "defaults"
+          end
```

Comment:
you can just make this more concisely:

```ruby
def default_mount_options
  node[:os] == "freebsd" ? "rw" : "defaults"
end
```

Reply:
Actually we think _only_ linux should be `defaults` and everything else is `rw`. So let's flip that logic around.

Reply:
@lamont-granquist, @thommay - The pull request has been updated.

Thanks for the help!